pkg/genericapiserver/options/server_run_options.go

```diff
@@ -304,6 +305,9 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 		"Per-resource etcd servers overrides, comma separated. The individual override "+
 		"format: group/resource#servers, where servers are http://ip:port, semicolon separated.")
 
+	fs.IntVar(&s.ExpectedRAMCapacityMB, "expected-ram-capacity-mb", s.ExpectedRAMCapacityMB,
+		"Expected RAM capacity for apiserver in MB (used to configure sizes of caches, etc.)")
```

Comment:
s/capacity/available/

..."limit" or "request" would be good words to work in, since they're how we talk about resource usage in config files...